src/features/chat/ui/ChatInput.tsx

```
@@ -1,39 +1,21 @@
   return (
     <div>
       <input
-        onChange={(e: ChangeEvent<HTMLInputElement>) => setValue(e.target.value)}
+        onChange={(e: ChangeEvent<HTMLInputElement>) => handleChangeMessage(e)}
         value={value}
+        className="border"
+        placeholder="채팅을 입력해주세요"
       />
-      <button onClick={handleSubmitChat}>채팅보내기</button>
+      <button onClick={createMessage}>채팅보내기</button>
     </div>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)